source/CardGames.py

```diff
@@ -422,7 +424,11 @@ def check_illegal_card(self):
         else:
           #Check if hideout 
           if ((card - previous_hideout) <= 3) and ((card - previous_hideout) != 0):
-            self.cards_in_play.append(card)
+            real_cards = []
+            for x in self.fugitive_deck:
+              if x.value == card:
+                self.cards_in_play.append(x)
```

Comment:
Are you able to append the card to self.cards_in_play and remove it from self.fugitive_deck in the same if statement?

Reply:
Done! Good catch! Code is simplified.